scene/resources/texture.h

```diff
@@ -445,8 +445,8 @@ class StreamTextureLayered : public TextureLayered {
 
 	enum FormatBits {
 		FORMAT_MASK_IMAGE_FORMAT = (1 << 20) - 1,
-		FORMAT_BIT_LOSSLESS = 1 << 20,
-		FORMAT_BIT_LOSSY = 1 << 21,
+		//FORMAT_BIT_LOSSLESS = 1 << 20,
```

Comment:
Why comment and not remove entirely?

Reply:
I remember the surrounding code doing that too, but apparently I misremembered.
I guess my intent was that the flags won't be reused for another purpose for compatibility.
But I suppose that doesn't matter for 4.0 anyway...
I'll have to check where the other bits are used, if at all.